plugin/bufsize/README.md

```diff
@@ -0,0 +1,34 @@
+# bufsize
+## Name
+*bufsize* - modifies EDNS0 buffer size for protecting answers from vulnerability of IP flagmentation.
+
+## Description
+*bufsize* can limit requester's UDP payload size.  
+It prevents IP fragmentation so that to be ready for the DNS Flag Day 2020 and to deal with DNS vulnerability.
```

Comment:
I would drop the mention of 2020 flagday here as it's not that relevant for the user.

Reply:
ok, I agree with that. I'll drop it.